etcdctl/ctlv3/command/printer.go

```diff
@@ -174,19 +174,29 @@ func makeMemberListTable(r v3.MemberListResponse) (hdr []string, rows [][]string
 }
 
 func makeEndpointStatusTable(statusList []epStatus) (hdr []string, rows [][]string) {
-	hdr = []string{"endpoint", "ID", "version", "db size", "is leader", "raft term", "raft index", "raft applied index", "errors"}
+	hdr = []string{"endpoint", "health", "error", "ID", "version", "db size", "is leader", "raft term", "raft index", "raft applied index", "resp errors"}
```

Comment:
1. let's remove `health` field
2. just add another extra field `resp error` at the end
3. and we can still get member ID of failed node from `endpointsFromCluster` (it only gets client URLs for now, but it can be extended to fetch member ID as well, and feed that into this response)